docs/USAGE.md

```diff
@@ -29,26 +33,61 @@ followed a manual installation as per [here](INSTALL.md)).
 
    j. [What is the Gap Limit](#gaplimit)
 
-3. [Try out a coinjoin; using sendpayment.py](#try-coinjoin)
+4. [Try out a coinjoin; using sendpayment.py](#try-coinjoin)
+
+5. [Running a "Maker" or "yield generator"](#run-maker)
+
+6. [Running the tumbler script to boost privacy of owned coins](#run-tumbler)
+
+<a name="data" />
+
+## Managing your Joinmarket data
 
-4. [Running a "Maker" or "yield generator"](#run-maker)
+First thing to do: go into `scripts/`, and run:
+
+        (jmvenv)$ python wallet-tool.py generate
 
-5. [Running the tumbler script to boost privacy of owned coins](#run-tumbler)
+This *should* quit with an error, because the connection to Bitcoin Core is not configured; we'll cover that in the next section.
+However, this first run will have automatically created a data directory.
+Locate the newly created file `joinmarket.cfg` which will be in your user home directory under `.joinmarket/`.
+So on Linux you should find it under `/home/username/.joinmarket/joinmarket.cfg`, and similarly for MacOS and Windows.
```

Comment:
Isn't it `/Users/username/Library/Application support/joinmarket/` on macOS, so kinda different and worth mentioning? But somebody more experienced with mac's could comment here.

Reply:
Yes, I didn't mention it because I wasn't exactly sure the right string; "similarly" is sufficiently vague to cover that :) I'll leave it like this until someone confirms the exactly right version.